### ISSUE TITLE
Unary Operator visitor not functional?

### ISSUE THREAD

**Issue by notzortex** [Kampfkarren/full-moon#84]
I've been trying to write a lua compiler utilizing full moon for ast parsing, and then using the visitors to go through the ast tree to emit bytecode but I've noticed that the visit_un_op Visitor member doesn't get called even when the parsed code contains an unary operator (-, !, #) 

example:
```rust
impl<'ast> Visitor<'ast> for CompileVisitor {
    fn visit_un_op(&mut self, node: &ast::UnOp<'ast>) {
        println!("{:?}", node); // never gets called
    }
}
...
visitor.visit_ast(&full_moon::parse("local x = #{4,3,2,1}").unwrap());

```

**Comment by Kampfkarren** [comment]
Will look into this today.

**Comment by Kampfkarren** [comment]
Fixed and pushed on 0.6.1.
